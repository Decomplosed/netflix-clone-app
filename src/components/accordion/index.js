import React from 'react';

export default function Accordion({ children, ...restProps }) {
  return <Container></Container>;
}
