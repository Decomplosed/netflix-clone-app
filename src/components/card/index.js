import React from 'react';

export const FeatureContext = createContext();
