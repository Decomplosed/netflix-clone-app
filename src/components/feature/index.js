import React from 'react';
import { Container, Title, SubTitle } from './styles/feature';

export default function Feature({ children, ...restProps }) {
  return <Container></Container>
}
