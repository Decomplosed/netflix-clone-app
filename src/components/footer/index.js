import React from 'react';

export default function Footer({ children, ...restProps }) {
  return <Container {...restProps}></Container>;
}
