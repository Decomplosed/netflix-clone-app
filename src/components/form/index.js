import React from 'react';
import {
  Container,
  Base,
  Error,
  Title,
  Text,
  TextSmall,
  Link,
  Input,
  Submit,
} from './styles/form';

export default function Form({ children, ...restProps }) {
  return <Container {...restProps}>{children}</Container>;
}

Form.Base = function FormBase({ children, ...restProps }) {
  return <Base {...restProps}>{children}</Base>;
};

Form.Error = function FormError({ children, ...restProps }) {
  return <Error {...restProps}>{children}</Error>;
};

Form.Title = function FormTitle({ children, ...restProps }) {
  return <Title {...restProps}>{children}</Title>;
};
