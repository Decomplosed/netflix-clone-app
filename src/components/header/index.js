import React from 'react';

export default Header({bg = true, children, ...restProps}) {}