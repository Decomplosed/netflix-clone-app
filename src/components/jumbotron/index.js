import React from 'react';

export default Jumbotron = () => {
  return <p>I am jumbotron</p>;
};
