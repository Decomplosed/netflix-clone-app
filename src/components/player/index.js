import React, { useState, useContext, createContext } from 'react';
import ReactDOM from 'react-dom';
