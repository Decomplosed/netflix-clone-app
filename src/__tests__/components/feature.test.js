import React from 'react';
import { render } from '@testing-library/react';
import { Feature } from '../../components';

describe('<Feature />', () => {
  it('Renders the <Feature /> with populated data', () => {
    const { container, getByText } = render();
  });
});
