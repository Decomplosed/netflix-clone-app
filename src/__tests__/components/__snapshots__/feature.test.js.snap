// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Feature /> Renders the <Feature /> with populated data 1`] = `
<div
  class="feature__Container-sc-1y24gor-0 hpCPBF"
>
  <h1
    class="feature__Title-sc-1y24gor-1 cVIPdK"
  >
    Unlimited films, TV programmes and more.
  </h1>
  <h2
    class="feature__SubTitle-sc-1y24gor-2 JryHG"
  >
    Watch anywhere. Cancel at any time.
  </h2>
</div>
`;
